src/Reflex/Spider/Internal.hs

```diff
@@ -452,50 +447,40 @@ newSubscriberFan subscribed = return $ Subscriber
       _ <- DMap.traverseWithKey f $ DMap.intersectionWithKey (\_ -> Pair) a subs --TODO: Would be nice to have DMap.traverse_
       return ()
   , subscriberInvalidateHeight = \old -> do
-      when debugInvalidateHeight $ putStrLn $ "invalidateSubscriberHeight: SubscriberFan" <> showNodeId subscribed
```

Comment:
we shouldn't actually need to lose these.

Reply:
oh, i see, we don't ... but we have them on all the time now?  not sure that's great either ...